includes/api/class-wc-rest-network-orders-controller.php

```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * REST API Network Orders controller
+ *
+ * Handles requests to the /orders/network endpoint
+ *
+ * @author   WooThemes
+ * @category API
+ * @package  WooCommerce/API
+ * @since    @todo
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * REST API Network Orders controller class.
+ *
+ * @package WooCommerce/API
+ * @extends WC_REST_Orders_Controller
+ */
+class WC_REST_Network_Orders_Controller extends WC_REST_Orders_Controller {
```

Comment:
@claudiosanches Are you happy for this to be added in v2 api or should we punt the feature until v3 api is introduced? It's more of an internal API.

Reply:
It's very small and for internal use, we could include and not even include into the docs for now.

Reply:
Not to mention that v3 will take a long time to be released, there is a lot of work there and things to be reviewed.